app/controllers/tagging_spreadsheets_controller.rb

```diff
@@ -15,7 +15,7 @@ def create
     if tagging_spreadsheet.valid?
       tagging_spreadsheet.save!
       InitialTaggingImport.perform_async(tagging_spreadsheet.id)
-      redirect_to tagging_spreadsheets_path
+      redirect_to tagging_spreadsheet, success: "Spreadsheet uploaded. It will be imported within a few minutes."
```

Comment:
Since we already use that pattern for error messages (check config/locales/en.yml), I thought it would be nice to continue doing that. It also improves the readability of the code and makes it easier to add longer and more self a explanatory messages. Just a thought. Though.